### PR TITLE
Remove bugzilla from serverless repositories.

### PR DESCRIPTION
**Pull request by markusthoemmes**
As per title. We're not using Bugzilla (yet).

### CODE REVIEW

**Comment by hongkailiu** [comment]
Need to fix this too.

https://github.com/openshift/release/blob/ebc6d9f94911cb85b382dee27aa5f79d883771d1/core-services/prow/02_config/_config.yaml#L882

**Comment by markusthoemmes** [comment]
@hongkailiu I adjusted accordingly I think. Not sure if we also need to drop the plugin. Dropping just the labels might be enough.

File: core-services/prow/02_config/_config.yaml [hunk 1]
```diff
@@ -1121,6 +1112,13 @@ tide:
     - openshift/tektoncd-pipeline-operator
     - openshift/tektoncd-catalog
     - openshift/tektoncd-triggers
+    - openshift/knative-build
```
Comment (matzew) [inline core-services/prow/02_config/_config.yaml]: do we really need knative-build ? :-)  /cc @vdemeester

Comment (vdemeester) [reply core-services/prow/02_config/_config.yaml]: nop knative-build is no more, I feel this repository should be archived, shouldn't it ?

**Pull request merged**